RileyLinkKit/UI/RileyLinkDeviceTableViewController.swift

```diff
@@ -15,7 +15,7 @@ public class RileyLinkDeviceTableViewController: UITableViewController, TextFiel
 
     public var device: RileyLinkDevice!
     
-    var rssiFetchTimer: NSTimer!
+    var rssiFetchTimer: NSTimer?
```

Comment:
Use `didSet` to invalidate, to make sure you're always calling `invalidate` before replacing the reference.